src/libraries/Microsoft.VisualBasic.Core/ref/Microsoft.VisualBasic.Core.cs

```diff
@@ -720,68 +723,70 @@ namespace Microsoft.VisualBasic.CompilerServices
     [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
     public sealed partial class CharArrayType
     {
         internal CharArrayType() { }
-        public static char[] FromObject(object Value) { throw null; }
-        public static char[] FromString(string Value) { throw null; }
+        public static char[] FromObject(object? Value) { throw null; }
+        public static char[] FromString(string? Value) { throw null; }
     }
     [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
     public sealed partial class CharType
     {
         internal CharType() { }
-        public static char FromObject(object Value) { throw null; }
-        public static char FromString(string Value) { throw null; }
+        public static char FromObject(object? Value) { throw null; }
+        public static char FromString(string? Value) { throw null; }
     }
     [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
     public sealed partial class Conversions
     {
         internal Conversions() { }
-        public static object ChangeType(object Expression, System.Type TargetType) { throw null; }
+        public static object? ChangeType(object? Expression, System.Type TargetType) { throw null; }
         [System.ObsoleteAttribute("do not use this method", true)]
         public static object FallbackUserDefinedConversion(object Expression, System.Type TargetType) { throw null; }
         public static string FromCharAndCount(char Value, int Count) { throw null; }
         public static string FromCharArray(char[] Value) { throw null; }
         public static string FromCharArraySubset(char[] Value, int StartIndex, int Length) { throw null; }
-        public static bool ToBoolean(object Value) { throw null; }
-        public static bool ToBoolean(string Value) { throw null; }
-        public static byte ToByte(object Value) { throw null; }
-        public static byte ToByte(string Value) { throw null; }
-        public static char ToChar(object Value) { throw null; }
-        public static char ToChar(string Value) { throw null; }
-        public static char[] ToCharArrayRankOne(object Value) { throw null; }
-        public static char[] ToCharArrayRankOne(string Value) { throw null; }
-        public static System.DateTime ToDate(object Value) { throw null; }
-        public static System.DateTime ToDate(string Value) { throw null; }
+        public static bool ToBoolean(object? Value) { throw null; }
+        public static bool ToBoolean(string? Value) { throw null; }
+        public static byte ToByte(object? Value) { throw null; }
+        public static byte ToByte(string? Value) { throw null; }
+        public static char ToChar(object? Value) { throw null; }
+        public static char ToChar(string? Value) { throw null; }
+        public static char[] ToCharArrayRankOne(object? Value) { throw null; }
```

Comment:
if value is null it returns an empty array?

Reply:
Yup:
https://github.com/dotnet/runtime/blob/ab56e22c299116493631fa4301f552c4b9a20605/src/libraries/Microsoft.VisualBasic.Core/src/Microsoft/VisualBasic/CompilerServices/Conversions.vb#L2104-L2112

Reply:
ok, thanks.